packages/reassure-measure/src/config.ts

```diff
@@ -1,21 +1,32 @@
-import type { render as RntlRender } from '@testing-library/react-native';
+import type { render as RntlRender, cleanup as RntlCleanup } from '@testing-library/react-native';
+import type { render as RtlRender, cleanup as RtlCleanup } from '@testing-library/react';
 
 let render;
+let cleanup;
 
 try {
   // eslint-disable-next-line import/no-extraneous-dependencies
   render = require('@testing-library/react-native').render;
+  // eslint-disable-next-line import/no-extraneous-dependencies
+  cleanup = require('@testing-library/react-native').cleanup;
 } catch {
-  // TODO add web support
-  throw new Error('Reassure requires `@testing-library/react-native` to be installed');
+  try {
+    // eslint-disable-next-line import/no-extraneous-dependencies
+    render = require('@testing-library/react').render;
+    // eslint-disable-next-line import/no-extraneous-dependencies
+    cleanup = require('@testing-library/react').cleanup;
+  } catch {
+    throw new Error('Reassure requires `@testing-library/react-native` or ``@testing-library/react-` to be installed');
```

Comment:
We should allow providing custom `render` + `cleanup` implementation. If user provides his own `render` + `cleanup` they should take precedence over auto-discovered ones. This probably means we have to move the `render` check into `measurePerformance`/`measureRender` call.

Reply:
👍